src/vs/workbench/services/configurationResolver/common/variableResolver.ts

```diff
@@ -270,6 +282,10 @@ export class AbstractVariableResolverService implements IConfigurationResolverSe
 					case 'workspaceFolderBasename':
 						return paths.basename(this.fsPath(getFolderUri()));
 
+					case 'userHome': {
+
```

Comment:
Looks like the implementation is missing here.

Reply:
Yeah, that was because of the async issue from before. I added it now.